op-batcher/batcher/espresso.go

```diff
@@ -246,6 +291,30 @@ const VERIFY_RECEIPT_TIMEOUT = 4 * time.Second
 // retrying a job that failed to verify the receipt.
 const VERIFY_RECEIPT_RETRY_DELAY = 100 * time.Millisecond
 
+// Evaluate the verification job.
+func evaluateVerification(jobResp espressoVerifyReceiptJobResponse) JobEvaluation {
+	err := jobResp.err
+
+	// If there's no error, continue handling the verification.
+	if err == nil {
+		return Handle
+	}
+
```

Comment:
Could not we have the same as here https://github.com/EspressoSystems/optimism-espresso-integration/pull/161/files#diff-b05a34d7c82ce2f74eec654c8523b20893001128c934781377efb6b091cd8219R229?